source/org/zfin/nomenclature/repair/GenotypeFeatureNameComparator.java

```diff
@@ -0,0 +1,29 @@
+package org.zfin.nomenclature.repair;
+
+import java.util.Comparator;
+import java.util.List;
+
+public class GenotypeFeatureNameComparator implements Comparator<GenotypeFeatureName> {
+
+    @Override
+    public int compare(GenotypeFeatureName pattern1, GenotypeFeatureName pattern2) {
+        if (pattern1.containsTransgenics() && !pattern2.containsTransgenics()) {
+            return 1;
+        } else if (pattern2.containsTransgenics() && !pattern1.containsTransgenics()) {
+            return -1;
+        }
+        return String.CASE_INSENSITIVE_ORDER.compare(pattern1.toString(), pattern2.toString());
+    }
+
+    public static boolean listsEqual(List<GenotypeFeatureName> list1, List<GenotypeFeatureName> list2) {
+        if (list1.size() != list2.size()) {
+            return false;
+        }
+        for(int i = 0; i < list1.size(); i++) {
```

Comment:
Could use this library:
CollectionUtils.isEqualCollection(list1, list2)

Reply:
Good point. I replaced that function so as not to reinvent the wheel. (Had to use ListUtils.isEqualList though since order is important here.)